src/accelerate/notebook_launcher.py

```diff
@@ -39,6 +40,8 @@ def notebook_launcher(function, args=(), num_processes=None, use_fp16=False, use
             the number of GPUs available otherwise.
         use_fp16 (:obj:`bool`, `optional`, defaults to :obj:`False`):
             If :obj:`True`, will use mixed precision training on multi-GPU.
```

Comment:
You can remove this from the documentation (we try to avoid documenting deprecated args so users don't start using them).